apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -57,16 +57,38 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const { rescheduleUid } = context.query;
   const { ssrInit } = await import("@server/lib/ssr");
   const ssr = await ssrInit(context);
-  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(context.req.headers.host ?? "");
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(
+    context.req.headers.host ?? "",
+    context.query.orgSlug
```

Comment:
Making sure we respect the way of accessing an org link using `/org/orgSlug` as well